helm-chart/harvester/README.md

```diff
@@ -0,0 +1,39 @@
+# Getting started
+If you want to play around with helm locally, you need to install it:
+- https://helm.sh/docs/intro/install/
+
+However, unless you are already running a kube cluster locally, it probably won't be very useful.
+In order to use a remote GCP cluster, you need to install the gcloud tool as well:
+- https://cloud.google.com/sdk/docs/install
+
+Remember to authenticate and point gcloud to your GKE cluster:
+`gcloud auth login`
+`gcloud container clusters get-credentials YourClusterName --region YourClusterRegion --project YourGCPProjectID`
+
+## How the Harvester helm-chart works
+### Kubernetes manifests
+If you check out our [templates](./templates) you'll find 4 resources:
+`deployment.yaml` - responsible for bringing up the pod hosting our app container. This actively use our docker image from ghcr.
+`service.yaml` - Binds to our deployment, exposing our app on a pre-determined port
+`ingress.yaml` - Binds to our service, directing traffic from outside the cluster to our app
+`storage-pvc.yaml` - Creates a persistent volume which the app uses to store SQL data
+
+### Values.yaml
+This is where we specify values referenced by our templates.
+Any value specified in [values.yaml](./values.yaml) is subject to change - please read the file to find available options.
+
+### Chart.yaml
+Name of the helm chart and optional versioning.
+We currently don't use the helm builtin versioning in this file, so these can be ignored for now.
+
+## Great! That's harvester, but can I use this to spawn other projects on our GKE cluster?
```

Comment:
At some point we may want to create some sort of template repo for this information.